exercises/isogram/tests/isogram.rs

```diff
@@ -0,0 +1,62 @@
+extern crate isogram;
+
+use isogram::check;
+
+#[test]
+fn empty_string() {
+    assert_eq!(check(""), true, "An empty string is a isogram.")
```

Comment:
can you confirm whether it is better to use `assert_eq!(something, true, message)` rather than `assert!(something, message)` ? Which is better, and why?

Reply:
I would prefer the ```assert_eq!``` because the output when the assertion fails is more informative than with the ```assert!``` macro.

With ```assert_eq!```:
![grafik](https://user-images.githubusercontent.com/17926729/31741910-44114d14-b456-11e7-9575-e0ef49015b4a.png)


With ```assert!```:
![grafik](https://user-images.githubusercontent.com/17926729/31741939-5a8950c8-b456-11e7-8131-243d54f40459.png)

Is there an advantage i missed to prefer one over the other?

Reply:
> I would prefer the `assert_eq!` because the output when the assertion fails is more informative than with the `assert!` macro.

> with `assert!`
> `panicked at '"eleven" has more than one 'e''`

It's true that it's unclear, but perhaps it could be mdae clear by changing the message to `"eleven has more than one 'e' and thus is not a pangram`.

I have decided that it doesn't matter to me what choice you make, only that I see the reasoning for it. Therefore, if I see information that may alter that reasoning, I am obligated to say it.

Reply:
I don't find the `assert_eq` much more helpful because there is no indication of which of the two (true/false) are the observed/expected values, unless there is some convention going on that I'm not aware of (such as "right" being the "right"/"correct" answer and/or the explanation being next to the "right" line).

If I'm working with booleans, I think I will be able to tell without assistance that there are only two values and that my test must be failing because I'm producing the wrong value and the correct value is the only other possibility. However, maybe we would like to contend this is not as easy for a newcomer?

Reply:
I think it depends on the level of the person working at this exercise. For people which are new in rust a kind of "verbose" output with more information is more helpfull. If I could choose how somebody should implement test for me being new to rust i would prefer the ```assert_eq!```.